### ISSUE TITLE
Job completion not happening or traceable

### ISSUE THREAD

**Issue by ghareesh** [joblib/joblib#1032]
Tring to run below command
train_scores, test_scores = validation_curve(RandomForestClassifier(verbose=1), 
                                             X, y, 
                                             param_name="n_estimators", 
                                             param_range=param_range,
                                             cv=3, 
                                             scoring="accuracy",n_jobs=-1)

/usr/local/lib/python3.6/site-packages/joblib/externals/loky/reusable_executor.py:193: UserWarning:
Trying to resize an executor with running jobs: waiting for jobs completion before resizing.

Dataset size is 68732 rows * 381 columns

The process keeps running for more than 1 Hour without any error, but no outputs. 

Is there a way to fasten this, 
I tried adding "with parallel_backend('threading', n_jobs=-1):" ,  but still no use

Regards
Hareesh

**Comment by ogrisel** [comment]
Hi @ghareesh, sorry for the slow reply.

I have tried to turn your code snippet into a [Minimal Reproducing Example](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) by including import statements, random data and some hyperparameters as follows:

```python
from sklearn.ensemble import RandomForestClassifier
from sklearn.datasets import make_classification
from sklearn.model_selection import validation_curve


X, y = make_classification(n_samples=68732, n_features=381, random_state=0)

train_scores, test_scores = validation_curve(RandomForestClassifier(verbose=1),
                                             X, y,
                                             param_name="n_estimators",
                                             param_range=[5, 10, 50],
                                             cv=3,
                                             scoring="accuracy",n_jobs=-1)
```

However I cannot reproduce the problem with scikit-learn and joblib master branches. Could you please confirm that you can reproduce the problem with the code above or a variation of it? Which versions of scikit-learn and joblib are you running?
